drivers/media/i2c/imx296.c

```diff
@@ -256,6 +258,36 @@ static int imx296_write(struct imx296 *sensor, u32 addr, u32 value, int *err)
 	return ret;
 }
 
+/*
+ * The supported formats.
+ * This table MUST contain 4 entries per format, to cover the various flip
+ * combinations in the order
+ * - no flip
+ * - h flip
+ * - v flip
+ * - h&v flips
+ */
+static const u32 mbus_codes[] = {
+	/* 10-bit modes. */
+	MEDIA_BUS_FMT_SRGGB10_1X10,
+	MEDIA_BUS_FMT_SGRBG10_1X10,
+	MEDIA_BUS_FMT_SGBRG10_1X10,
+	MEDIA_BUS_FMT_SBGGR10_1X10,
+};
+
+static u32 imx296_mbus_code(struct imx296 *sensor)
+{
+	unsigned int i = 0;
+
```

Comment:
The imx219 version of this function includes
```
lockdep_assert_held(&imx219->mutex);
```
as this function needs to be called with the mutex held

Reply:
Hmmm, do I need this to guard against changing h/v flip ctrls?  I can't see any other reason for it...